src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -127,6 +127,7 @@
       <!-- TODO: set this from some user-facing property?  -1 means use the default baked into dotnet.native.js -->
       <_WasmPThreadPoolInitialSize Condition="'$(_WasmPThreadPoolInitialSize)' == ''">-1</_WasmPThreadPoolInitialSize>
       <_WasmPThreadPoolUnusedSize Condition="'$(_WasmPThreadPoolUnusedSize)' == ''">-1</_WasmPThreadPoolUnusedSize>
+      <_WasmIsPublishing Condition="'$(_WasmIsPublishing)' == '' and '$(_IsPublishing)' != ''">$(_IsPublishing)</_WasmIsPublishing>
```

Comment:
Why do we have `_WasmIsPublishing`? `_IsPublishing` alone looks sufficient. We're not using it anywhere else than this PR, so I would vote for leaving only the generic-named property. Then we can also simplify change in src/mono/wasm/build/WasmApp.Common.targets.

Reply:
The problem is with nested publish https://github.com/dotnet/runtime/blob/fe15a5b55b1051fdc78180dfd318a09d7a105318/src/mono/wasm/build/WasmApp.Common.targets#L449. It is explicitly passing `_IsPublishing` as empty value to work with the rest of the SDK. So I'm adding a wasm specific property. Here I'm just ready the standard one in case the special one is empty (probably not needed, just in case)